### DISCUSSION TITLE
blacklist and whitelist

### DISCUSSION THREAD

**Discussion by wangwentong-lunaon** [pmndrs/zustand#2439] [Q&A]
persist is not support blacklist and whitelist?

**Comment by dai-shi** [comment]
Do you mean something like `partialize`?

**Reply by wangwentong-lunaon** [reply]
https://github.com/pmndrs/zustand/blob/v3.7.2/src/middleware/persist.ts
this version is support, not support from v4

**Reply by dai-shi** [reply]
yeah, it's replaced with `partialize`.
